bndtools.core/bndtools.shared.bndrun

```diff
@@ -72,8 +87,6 @@ crossplatform.bundles: \
 	javax.inject,\
 	javax.xml,\
 	javax.xml.stream,\
-	slf4j.api,\
-	slf4j.simple,\
```

Comment:
Why remove slf4j impl? Without some impl, the log messages are lost. (nop behavior is the default when no impl).